### PR TITLE
(port) Introduction of sweet Italian Liqueur, Amaretto

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr: https://github.com/tgstation/tgstation/pull/54466

adds 5 new drinks
amaretto
amaretto alexander
ginger amaretto
godfather
godmother
#### Changelog

:cl:  
rscadd: Added new drinks
/:cl:

### CODE REVIEW

**Review by Partheo — APPROVED** [review]
About time

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -815,4 +815,22 @@
...
+/datum/chemical_reaction/amaretto_alexander
+	results = list(/datum/reagent/consumable/ethanol/amaretto_alexander = 3)
+	required_reagents = list(/datum/reagent/consumable/ethanol/amaretto = 1, /datum/reagent/consumable/ethanol/creme_de_cacao = 1, /datum/reagent/consumable/cream = 1)
+
+/datum/chemical_reaction/ginger_amaretto
+	results = list(/datum/reagent/consumable/ethanol/ginger_amaretto = 4)
+	required_reagents = list(/datum/reagent/consumable/ethanol/amaretto = 1, /datum/reagent/consumable/sol_dry = 1, /datum/reagent/consumable/ice = 1, /datum/reagent/consumable/lemonjuice = 1)
+
+
+
```
Comment (JamieD1) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
Cornflakes @boodaliboo @Partheo

**Comment by boodaliboo** [comment]
?

**Comment by Partheo** [comment]
He means fix your merge conflicts @boodaliboo 
image

**Comment by boodaliboo** [comment]
? not sure why

**Comment by Partheo** [comment]
I've compared your icon file against live and I honestly can't tell what the issue is. Nothing seems to be amiss. Any idea @JamieD1 ?

**Comment by jcatjmeow** [comment]
Merge conflict almost fixed. Just gotta re-add some over-written drinks

**Pull request merged**